example/styleguide/0-Getting Started/0-Welcome.mdx

```diff
@@ -13,4 +13,4 @@ publishToStyleGuide: true
 1. Built-in support for showing [Storybook](https://storybook.js.org/) components.
 1. and more..
 
-[Next: Installation & Usage](/0__getting-started/1__installation-&-usage/)
+[Next: Installation & Usage](/getting-started/installation-&-usage/)
```

Comment:
I think all of these links need to be updated, since they still need to use the number prefix in the path.

Reply:
done